share/sysbuild/cmake/modules/sysbuild_extensions.cmake

```diff
@@ -2,6 +2,83 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+# Usage:
+#   load_cache(IMAGE <image> BINARY_DIR <dir>)
+#
+# This function will load the CMakeCache.txt file from the binary directory
+# given by the BINARY_DIR argument.
+#
+# All CMake cache variables are stored in a custom target which is identified by
+# the name given as value to the IMAGE argument.
+#
+# IMAGE:      image name identifying the cache for later sysbuild_get() lookup calls.
+# BINARY_DIR: binary directory (build dir) containing the CMakeCache.txt file to load.
+function(load_cache)
+  set(single_args APPLICATION BINARY_DIR)
```

Comment:
documentation here does not seem to match arguments, should `IMAGE` be `APPLICATION`?

Reply:
fixed.

I changed from APP.. --> IMAGE to have the functions better aligned, but seems I forgot to do it allover the place.
Thanks for spotting.